docs/interfaces/data_product_hub/exploration.md

```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
 
 ### **GraphQL Tab**
 
-The GraphQL tab provides an interface for advanced users and developers to write and execute GraphQL queries. GraphQL enables precise data retrieval, allowing full control over the structure of the returned data. This is useful for performing complex or customized data extractions that extend beyond the standard capabilities of the Studio interface.
+The GraphQL tab provides an interface for advanced users and developers to programmatically interact with the data model. It displays the data returned by a GraphQL query alongside the query itself. This feature is useful for performing complex or customized data extractions that go beyond the standard capabilities of the Studio interface.
+
+The interface includes syntax highlighting,  autocompletion and automatic documentation and error highlighting. Users can utilize GraphiQL to interactively build both page and static queries.
+
+The GraphQL tab is designed as follows:
+
+
+<center>
+<img src="/interfaces/data_product_hub/exploration/graphql.png" alt="DPH" style="width:40rem; border: 1px solid black;" />
+</center>
+
+
+The GraphQL tab is divided into three sections:
+
+1. The left sidebar contains buttons for opening plugin panes and changing settings.
+2. The middle section is used to write queries.
+3. The right section displays the results of the executed queries.
+
+The left sidebar is further divided into a top and bottom section. The top section provides buttons for accessing the documentation explorer and query history. The bottom section contains buttons to refetch the schema, view keyboard shortcuts, and modify settings.
+
+In the middle section, queries can be written using the following format:
+
+```graphql
+query LensQuery {
+    table{}
+}
+```
+
+Within the curly braces, press "Ctrl + Space" (or "Shift + Space" as an alternate keyboard shortcut) to open the autocomplete window. This will prompt a query for either logical tables or metrics in the data model. 
+
+<center>
+<img src="/interfaces/data_product_hub/exploration/graphql_autocomplete.png" alt="DPH" style="width:20rem; border: 1px solid black;" />
+</center>
+
+Construct a query like below:
+
+```
+query LensQuery {
+  table {
+    sales {
+      source
+    }
+  }
+}
+```
+
+**Components:**
```

Comment:
I think you have written these components in generic graphql terminology, write it in the context of DataOS